core/controllers/suggestion.py

```diff
@@ -388,7 +388,7 @@ def put(self, suggestion_id):
         Raises:
             InvalidInputException. The suggestion is already handled.
             InvalidInputException. The 'translation_html' parameter is missing.
-            InvalidInputException. The 'translation_html' parameter should be a
+            InvalidInputException. The 'translation_html' parameter is a
```

Comment:
is --> is not

(that's correct, right? This is supposed to describe the error case)

Reply:
Done